src/frontend/screens/Library/components/AlphabetFilter/AlphabetFilter.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useMemo, useContext, useEffect } from 'react'
+import './index.css'
+import { GameInfo } from 'common/types'
+import { themeType } from 'frontend/components/UI/ThemeSelector/index'
+import ContextProvider from 'frontend/state/ContextProvider'
+
+const ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('')
+const SYMBOLS = ['#']
+const ALPHABET_AND_SYMBOLS = [...ALPHABET, ...SYMBOLS]
+
+interface AlphabetFilterProps {
+  currentFilter: string | null
+  onFilterChange: (filter: string | null) => void
+  allGames: GameInfo[]
+}
+
+const AlphabetFilter: React.FC<AlphabetFilterProps> = ({
+  currentFilter,
+  onFilterChange,
+  allGames
+}) => {
+  const { theme } = useContext(ContextProvider)
+  useEffect(() => {
+    if (themeType[theme] === 'light') {
+      document.body.setAttribute('data-theme-light', 'true')
+    } else {
+      document.body.removeAttribute('data-theme-light')
```

Comment:
I don't think this is needed, but if we ever do need this, the `AlphabetFilter` component shouldn't be the one adding/removing attributes of the body element, components should only be concerned about it's internal state, not change the global scope